feat: add context processors to add new files

catalinaIlabaca opened the pull request:
Se agrega la extension del contexto para los certificados creada por thomas para agregar nuevos parámetros a los certificados

fsologureng reviewed (changes requested):
Mantener orden alfabético

fsologureng commented on requirements/apps.txt:
```diff
@@ -1,4 +1,5 @@
 -e git+https://github.com/eol-uchile/course_classification@0.2.0#egg=course_classification
+-e git+https://github.com/eol-uchile/context-extender@0.0.1#egg=context-extender
```
el tag es `0.0.1-dev1`

fsologureng commented:
Cambié las referencias en el propio paquete para que referencie mejor a sus dependencias. Cambiar al nuevo tag 0.0.2-dev1.

fsologureng reviewed (changes requested):
Cambié las referencias en el propio paquete para que referencie mejor a sus dependencias. Cambiar al nuevo tag 0.0.2-dev1.
Cambiarla tb para las dependencias asociadas en este pr.

catalinaIlabaca committed:
feat: add context processors to add new files

fsologureng committed:
build: changes tagged refs to zipped pkg & more
 - removes `-e` flag to pip to avoid editable mode to released packages.
 - update eol_sso_login to latest version.
 - update context-extender to devel version compatible w/eol_sso_login@0.0.3.
